### PR TITLE
change to the .format() function

### PR DESCRIPTION
**Pull request by MarkusHackspacher**
change to the .format() function.

### CODE REVIEW

File: horizons/ai/aiplayer/productionchain.py [hunk 1]
```diff
@@ -234,7 +238,8 @@ def __init__(self, settlement_manager, resource_id, production_line, abstract_bu
 
 	def assign_identifier(self, prefix):
 		"""Recursively assign an identifier to this subtree to know which subtree owns which resource quota."""
-		self.identifier = '%s/%d,%d' % (prefix, self.resource_id, self.abstract_building.id)
+		self.identifier = '{}/{:d},{:d}'.index(
```
Comment (squiddy) [inline horizons/ai/aiplayer/productionchain.py]: index? Also, that line break is unnecessary. As well as some others in this pull request.

Comment (MarkusHackspacher) [inline horizons/ai/aiplayer/productionchain.py]: Sorry, It mean format, it is fixed. without the line break is the line 101 long, have a tab 4 spaces.

**Comment by LinuxDonald** [comment]
@squiddy can it be now merged?

File: horizons/ai/aiplayer/mission/specialdomestictrade.py [hunk 2]
```diff
@@ -114,8 +118,12 @@ def _reached_source_settlement(self):
 			self.report_failure('No resources to transport')
 
 	def _move_to_destination_settlement(self):
-		self._move_to_warehouse_area(self.destination_settlement_manager.settlement.warehouse.position, Callback(self._reached_destination_settlement),
```
Comment (squiddy) [inline horizons/ai/aiplayer/mission/specialdomestictrade.py]: These changes make it unnecessary difficult to review changes. At the very least, I'd like to see these things broken up into 2 commits, format first, line length second.

Comment (MarkusHackspacher) [inline horizons/ai/aiplayer/mission/specialdomestictrade.py]: Let my explain my change: I make a new line after ( and , with | mark a new line:
```
		self._move_to_warehouse_area(self.source_settlement_manager.settlement.warehouse.position, Callback(self._reached_source_settlement),
                                             |                                                            |                                          |
			Callback(self._move_to_source_settlement), 'Unable to move to the source settlement (%s)' % self.source_settlement_manager.settlement.get_component(NamedComponent).name)
                                                                  |                                                |
```
Is it possible to accept this in this commit?

This construct is in
https://github.com/MarkusHackspacher/unknown-horizons/blob/71268cac0c4ab41e8ff21ae26784e1ed8a450119/horizons/ai/aiplayer/mission/internationaltrade.py#L94-L99

https://github.com/MarkusHackspacher/unknown-horizons/blob/71268cac0c4ab41e8ff21ae26784e1ed8a450119/horizons/ai/aiplayer/mission/internationaltrade.py#L128-L133

https://github.com/MarkusHackspacher/unknown-horizons/blob/71268cac0c4ab41e8ff21ae26784e1ed8a450119/horizons/ai/aiplayer/mission/internationaltrade.py#L173-L178

https://github.com/MarkusHackspacher/unknown-horizons/blob/71268cac0c4ab41e8ff21ae26784e1ed8a450119/horizons/ai/aiplayer/mission/specialdomestictrade.py#L79-L84

https://github.com/MarkusHackspacher/unknown-horizons/blob/71268cac0c4ab41e8ff21ae26784e1ed8a450119/horizons/ai/aiplayer/mission/specialdomestictrade.py#L121-L126

Comment (squiddy) [inline horizons/ai/aiplayer/mission/specialdomestictrade.py]: I think the change itself is fine, I'd just like to see this split up into two commits next time.

**Pull request merged**